docs/install.rst

```diff
@@ -70,6 +70,10 @@ need to install Python 3 with virtualenv in your system as well.
 
         sudo apt-get install redis-server
 
+.. note::
+
+   Some unit tests depend on Hg (mercurial).
+   Be sure that Hg is `installed <https://www.mercurial-scm.org/wiki/Download>`_ if you want to see all the tests pass.
```

Comment:
I'm happy to jump in and help if its faster. If you want any guidance on this, I'm happy to pair as well -- feel free to drop by https://gitter.im/rtfd/readthedocs.org if you'd like to set something up